MobileCenterCrashes/MobileCenterCrashes/Internals/MSWrapperExceptionManager.m

```diff
@@ -89,7 +89,7 @@ + (NSString*)getDataFilenameWithUUIDRef:(CFUUIDRef)uuidRef {
 }
 
 + (BOOL) isDataFile:(NSString*)path {
-  return path ? [path hasSuffix:[@"" stringByAppendingPathExtension:[self dataFileExtension]]] : false;
+  return path ? [path hasSuffix:[@"." stringByAppendingString:[self dataFileExtension]]] : false;
```

Comment:
Sorry @achocron, you were asked to add null check for `path` in the last PR but you don't need this. Feel free to remove.